canary/consumer-java/src/main/java/com/amazon/kinesis/video/canary/consumer/WebrtcStorageCanaryConsumer.java

```diff
@@ -0,0 +1,215 @@
+package com.amazon.kinesis.video.canary.consumer;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.FutureTask;
+import java.util.Date;
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.ArrayList;
+import java.util.concurrent.Future;
+import java.lang.Exception;
+import java.text.MessageFormat;
+
+import com.amazonaws.auth.EnvironmentVariableCredentialsProvider;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.cloudwatch.AmazonCloudWatch;
+import com.amazonaws.services.cloudwatch.AmazonCloudWatchClientBuilder;
+import com.amazonaws.services.kinesisvideo.AmazonKinesisVideo;
+import com.amazonaws.services.kinesisvideo.AmazonKinesisVideoClientBuilder;
+import com.amazonaws.services.kinesisvideo.model.APIName;
+import com.amazonaws.services.kinesisvideo.model.GetDataEndpointRequest;
+import com.amazonaws.services.kinesisvideo.model.TimestampRange;
+import com.amazonaws.services.kinesisvideo.model.FragmentSelector;
+import com.amazonaws.services.cloudwatch.model.Dimension;
+import com.amazonaws.services.cloudwatch.model.MetricDatum;
+import com.amazonaws.services.cloudwatch.model.PutMetricDataRequest;
+import com.amazonaws.services.cloudwatch.model.StandardUnit;
+import com.amazonaws.services.kinesisvideo.model.StartSelector;
+import com.amazonaws.services.kinesisvideo.model.StartSelectorType;
+import com.amazonaws.kinesisvideo.parser.utilities.FrameVisitor;
+import com.amazonaws.kinesisvideo.parser.examples.GetMediaWorker;
+
+import lombok.extern.log4j.Log4j2;
+
+/*
+ * Canary for WebRTC with Storage thro Media Server
+ * 
+ * For longRun-configured jobs, this Canary will emit FragmentContinuity metrics by continuously
+ * checking for any newly ingested fragments for the given stream. The fragment list is checked
+ * for new fragments every "fragmentDuration * 2" The fragmentDuration is set by the encoder in
+ * configuration in Media Server.
+ * 
+ * For periodic-configured jobs, this Canary will emit TimeToFirstFragment metrics by continuously
+ * checking for consumable media from the specified stream via GetMedia calls.
+ */
+
+@Log4j2
+public class WebrtcStorageCanaryConsumer {
+    protected static Date canaryStartTime;
```

Comment:
Ok I moved the definitions of some of these up from main to be able to make them `final`.

Some are `protected` so that they may be exposed to the RealTimeFrameProcessor class.

They are `static` so that they can be referenced from static contexts in main(). I think we've previously discussed how this would be necessary if we want to avoid extrapolating WebrtcStorageCanaryConsumer to yet another new, separate class, and if they were to be declared in main(), the static functions here would need very long lists of parameters.

I guess making one more new class to solve all this is no big deal, but I wanted to avoid cluttering the existing Canary with new files which are only relevant to the storage Canary. How should we move forward?